src/lib/pages/query/components/QueryArea.tsx

```diff
@@ -163,10 +164,17 @@ export const QueryArea = ({
         </Box>
         <Spacer />
         <Box w="full">
-          <JsonReadOnly topic="Return Output" text={res} height="240px" />
-          <Flex justifyContent="flex-end" gap={2}>
-            <CopyButton isDisable={res.length === 0} value={res} />
-          </Flex>
+          <JsonReadOnly
+            topic="Return Output"
+            text={res}
+            canCopy={res.length !== 0}
+          />
+          {/* If response line count > 100, the copy button is visible. */}
+          {jsonLineCount(res) > 100 && (
+            <Flex justifyContent="flex-end" mt={2}>
```

Comment:
mt={4} to match the left side
<img width="1210" alt="Screen Shot 2566-03-16 at 15 52 04" src="https://user-images.githubusercontent.com/52099595/225564453-4bebb260-23f5-4e5c-b8b2-c2fe1f7ff099.png">